internal/plugin/server.go

```diff
@@ -157,7 +157,7 @@ func (plugin *NvidiaDevicePlugin) Start() error {
 		// TODO: add MPS health check
 		err := plugin.rm.CheckHealth(plugin.stop, plugin.health)
 		if err != nil {
-			klog.Infof("Failed to start health check: %v; continuing with health checks disabled", err)
+			klog.Errorf("Failed to start health check: %v; continuing with health checks disabled", err)
```

Comment:
This could also be a warning instead, but I don't think it's a blocker.

Reply:
Thanks for your reply :)